### PR TITLE
Fixed several grammar errors and typos

### PR DESCRIPTION
**Pull request by AndrewDawes**
**Description**
I found some grammar errors and typos in the documentation and have fixed them. Maintaining documentation is something that's important to me :-)

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/tutorial/index.rst [hunk 1]
```diff
@@ -53,7 +53,7 @@ Enjoy your exploration of the CodeIgniter framework.
 Getting Up and Running
 **********************
 
-Installing CodeIgnier
+Installing CodeIgniter
 =====================
```
Comment (kenjis) [inline user_guide_src/source/tutorial/index.rst:57]:
```suggestion
Installing CodeIgniter
======================
```

**Review by kenjis — APPROVED** [review]
Thank you!

**Review by kenjis — APPROVED** [review]

**Review by paulbalandan — APPROVED** [review]

File: user_guide_src/source/installation/repositories.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ There are several development repositories, of interest to potential contributor
 +------------------+--------------+-----------------------------------------------------------------+
 | shield           | developers   | Authentication and Authorization Library for CodeIgniter 4      |
 +------------------+--------------+-----------------------------------------------------------------+
-| tasks            | developers   | Task Scheduler for CodeIgnter 4                                 |
+| tasks            | developers   | Task Scheduler for CodeIgniter 4                                 |
```
Comment (paulbalandan) [inline user_guide_src/source/installation/repositories.rst:31]:
```suggestion
| tasks            | developers   | Task Scheduler for CodeIgniter 4                                |
```

**Comment by paulbalandan** [comment]
Thank you, @AndrewDawes

**Pull request merged**